master/buildbot/test/unit/test_worker_ec2.py

```diff
@@ -22,8 +24,6 @@
     boto = None
     ec2 = None
 
-from twisted.trial import unittest
```

Comment:
why did you move the import?  lexicographically, `mock`, `moto`, `boto` and `ec2` come before `twisted`...